lib/lrama/counterexamples/path.rb

```diff
@@ -20,6 +20,10 @@ def to_s
         "#<Path(#{type})>"
       end
       alias :inspect :to_s
+
+      def type
+        raise NoMethodError
```

Comment:
It's expected for sub classes to implement `#type` then `NotImplementedError` is better than `NoMethodError`.